src/NServiceBus.AcceptanceTests/Tx/ImmediateDispatch/When_requesting_immediate_dispatch_using_scope_suppress.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace NServiceBus.AcceptanceTests.Tx
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using System.Transactions;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    //note: this test will no longer be relevant in v7
```

Comment:
I actually wanted that but the tests doesn't ref obsoletex so I got lazy :(  Will fix that